src/utils/hooks/useTextKeys.tsx

```diff
@@ -0,0 +1,36 @@
+import { TranslationsContext } from '@hedviginsurance/textkeyfy'
+import * as React from 'react'
+
+const placeholderRegex = new RegExp('({[a-zA-Z0-9_]+})', 'g')
+const placeholderKeyRegex = new RegExp('([a-zA-Z0-9_]+)', 'g')
+
+String.prototype.fill = function(replacements) {
```

Comment:
I didn't really understand what `fill` did at first, maybe call this `.withReplacements` or something? 😄

Reply:
I agree 👍